### ISSUE TITLE
tf.compat.v1.flags.EnumClassSerializer throws error on Windows

### ISSUE THREAD

**Issue by lugalUrim** [tensorflow/tensorflow#48864]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): TensorFlow 2.4.1
- Python version: 3.7.5
- CUDA/cuDNN version: N/a
- GPU model and memory: N/a

**Standalone code to reproduce the issue**
```
import tensorflow as tf
tf.compat.v1.flags.EnumClassSerializer
```
**Outputs:**
```
AttributeError: module 'tensorflow.python.platform.flags' has no attribute 'EnumClassSerializer'
```

**Describe the current behavior**
Throws AttributeError. The same problem occurs when I try to use the following APIs:
 - tf.compat.v1.app.flags.EnumClassListSerializer
 - tf.compat.v1.app.flags.EnumClassSerializer
 - tf.compat.v1.flags.EnumClassListSerializer

**Describe the expected behavior**
I can use these APIs on Linux. Expect them to work on Windows as well.

**Comment by tilakrayal** [comment]
@lugalUrim ,

I was able to run the given code without any errors.Please find the [gist](https://colab.research.google.com/gist/tilakrayal/60efd73f0e5bd11e7ba4e35e74e56932/48864.ipynb) here. Could you please try to execute the code in colab from your side and let us know if you are facing the issue again.

Thanks!

**Comment by bhack** [comment]
@lugalUrim Can you check if is it solved on your Windows system with TensorFlow 2.5.0.rc2? 
It seems running without problem under Wine:

```
docker run -it --rm tobix/pywine bash -c "wine python -m pip install tensorflow-cpu==2.5.0.rc2 && wine python -c 'import tensorflow as tf; print(tf.__version__); tf.compat.v1.flags.EnumClassSerializer'"
```
